src/Microsoft.Azure.ServiceBus/Management/ManagementClient.cs

```diff
@@ -0,0 +1,588 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Microsoft.Azure.ServiceBus.Core;
+using Microsoft.Azure.ServiceBus.Primitives;
+
+namespace Microsoft.Azure.ServiceBus.Management
+{
+    // TODO: Document all exceptions that might be thrown
+    // TODO: Retry for transient
+    public class ManagementClient : ClientEntity, IManagementClient
+    {
+        private bool ownsConnection;
+        private HttpClient httpClient;
+        private ServiceBusConnectionStringBuilder csBuilder;
+        private string namespaceConnectionString;
+        
+        // TODO: Expose other constructor overloads
+        public ManagementClient(ServiceBusConnectionStringBuilder connectionStringBuilder)
```

Comment:
Regarding ctor and TODO comment - management client shoudl by default receive connection string and internally exercise `ServiceBusConnectionStringBuilder`. While it's not that difficult to write
``` c#
new ManagementClient(new ServiceBusConnectionStringBuilder(connectionString));
```

customers will appreciate just doing

``` c#
new ManagementClient(connectionString);
```